docs/self-managed/reference-architecture/reference-architecture.md

```diff
@@ -0,0 +1,128 @@
+- **IT Managers**: To ensure the system meets business requirements and is maintainable.
+
+Key Benefits:
+
+- **Accelerated Deployment**: Predefined best practices and guidelines simplify the deployment process, reducing the time and effort required to set up a reliable workflow automation solution.
+- **Consistency**: Ensures consistency across deployments by standardizing system components and their configurations, which helps reduce the risk of errors and simplifies maintenance.
+- **Enhanced Security**: Reference architectures incorporate best practices for securing Camunda 8 deployments, ensuring that sensitive data and processes are protected through standard security measures like encryption, authentication, and access controls.
+
+## Customization and Flexibility
+
+It's important to note that reference architectures are not a one-size-fits-all solution. Each organization has unique requirements and constraints that may necessitate modifications to the provided blueprints. While these reference architectures offer a solid foundation and best practices, they should be adapted to fit the specific needs of your project. Use them as a starting point to start your Camunda 8 implementation process, but be prepared to make adjustments to ensure they align with your goals and infrastructure.
+
+## Support Considerations
+
+We recognize that deviations from the reference architecture are unavoidable. However, such changes will introduce additional complexity, making troubleshooting more difficult. When modifications are required, ensure they are well-documented to facilitate future maintenance and support more quickly. Camunda publishes [supported environments](/docs/reference/supported-environments/) information to help you navigate supported configurations.
+
+## Architecture
+
+<!-- TODO: include overview, Hamza had good pictures on this topic -->
+
+![Camunda Architecture](./img/placeholder.png)
+
+### Orchestration Cluster vs Web Modeler and Console
+
+When designing a reference architecture, it's essential to understand the differences between an orchestration cluster and Web Modeler and Console self-managed. These components play crucial roles in the deployment and operation of processes, but they serve different purposes and include distinct components.
+
+#### Orchestration Cluster
+
+We refer to the orchestration or automation cluster to the core of Camunda.
+
+The included components are:
+
+- [Zeebe](/components/zeebe/zeebe-overview.md): A workflow engine for orchestrating microservices and managing stateful, long-running business processes.
+- [Operate](/components/operate/operate-introduction.md): A monitoring tool for visualizing and troubleshooting workflows running in Zeebe.
+- [Tasklist](/components/tasklist/introduction-to-tasklist.md): A user interface for managing and completing human tasks within workflows.
+- [Optimize](#TODO): An analytics tool for generating reports and insights based on workflow data.
```

Comment:
@conceptualshark maybe you could reveal how to link to e.g. https://docs.camunda.io/optimize/next/components/what-is-optimize/

SInce Optimize is weirdly set up in the docs, I haven't figured out the linkage yet.

Reply:
You can sub in `$optimize$` for any `/optimize` links - so this one should look like `$optimize$/components/what-is-optimize/`.

Reply:
thanks a lot! 🙇